JKBTumpalan/evtf_project
# Feature/deployment changes

# Ticket
- None

# Changes
- Homepage links fix
- API calls to the EC2 endpoints
- Dockerfile updates for the API server

# Dependencies
None

# Notes
- None

# Test Plan
1. Pull branch
2. change working directory to **_/evtf_project_**
3. `docker-compose up --build`
- [ ] Check if the homepage links can now navigate to their respective pages
- [ ] Check if the classifiers work normally
